content/modules/redisearch/_index.md

```diff
@@ -7,145 +7,48 @@ categories: ["Modules"]
+The index knows how to index each field, but that's not enough.
+We need to actually store the data for retrieval.
+In a simple use case, you can just push documents to the database and RediSearch creates a complete index and document database.
 
-RediSearch differs from its open source brother in one key
-aspect. The enterprise version can work with multiple shards on a
-distributed cluster such as Redis Enterprise Software and the open
-source version is limited to using only one shard. When a RediSearch
-enabled database in Redis Enterprise Software is created, the number of
-partitions and shards the data is spread across is set.
+For storing documents, the RediSearch engine relies on Redis hashes,
+where each document is represented by a single key, and each property and its value by a hash key and element.
+For retrieval, you simply perform an HGETALL query on each retrieved document, returning its entire data.
+If the user needs to retrieve a specific document by its ID, a simple HGETALL can be performed by the user.
 
-It is critical to understand that **this value can only be configured
-when creating the database**. That makes it imperative in a production
-cluster to set this value according to how many shards you have. As
-of RediSearch Enterprise 1.0.1, the default is **PARTITIONS AUTO**. So
-the value is set by the cluster on your behalf and matched to the
-number of shards. Once you create the database, you cannot
-edit this value or the number of shards the database has. If at a later
-point you need to expand the database with more shards, you must create
-a new database with the settings you need, then [replicate the
-data]({{< relref "/rs/administering/designing-production/active-passive.md" >}})
-from the current database to the new one.
+## RediSearch in Active-Active databases
 
-If you need to create a database that has a PARTITIONS value that is not
-AUTO, this is not typical, you select the **Redis Modules** checkbox,
-then the settings icon, the alter the number of partitions.
+As a result of the new RediSearch architecture and methodology, [RediSearch 2.x supports Active-Active databases]({{< relref "/modules/redisearch/redisearch-active-active.md" >}}).
+You can now serve your index information from geo-distributed database instances.
 
-<!-- ![redisearch_partitions](/images/rs/redisearch_partitions.png) -->
+## Resharding and RediSearch
 
-## More information
+By moving the index out of the keyspace and structuring the data as hashes, RediSearch 2.x makes it possible to reshard the database.
+When half of the data moves to the new shard, the index related to that data is created synchronously and RediSearch removes the keys from the index when it detects that the keys were deleted.
+Because the index on the new shard is created synchronously though, it's expected that the resharding process will take longer than resharding of a database without RediSearch.
 
```

Comment:
Should we mention the knows issue that during the resharding searches are not possible?